checksum/crc/crc8.go

```diff
@@ -0,0 +1,53 @@
+// crc8.go
+// description: Calculate CRC8
+// details:
+// A cyclic redundancy check (CRC) is an error-detecting code commonly used in digital networks and storage devices to detect accidental changes to raw data. See more [CRC](https://en.wikipedia.org/wiki/Cyclic_redundancy_check)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see crc8_test.go
+
+package crc
```

Comment:
Package comment is missing. You need to include the package comment for our auto doc generator to work properly.